backend/epub_reader.py

```diff
@@ -0,0 +1,15 @@
+import epub
+
+book  =  epub . open_epub ( 'test.epub' ) 
```

Comment:
https://realpython.com/python-pep8/

There are a lot of spaces in between things, just make sure to clean that up

Reply:
This file was the old version that was in the backend folder, that I deleted sometime ago. I edited it in the new epub_reader.py file.